diff --git a/dist/server/lib/router-utils/setup-dev-bundler.js b/dist/server/lib/router-utils/setup-dev-bundler.js
index 2924899412660e0dfefe1bcecf6213f2e19a5c07..63f71fe7aae426907281380437968801e9b89647 100644
--- a/dist/server/lib/router-utils/setup-dev-bundler.js
+++ b/dist/server/lib/router-utils/setup-dev-bundler.js
@@ -705,6 +705,7 @@ async function startWatcher(opts) {
             serverFields.hasAppNotFound = hasRootAppNotFound;
             opts.fsChecker.middlewareMatcher = ((_serverFields_middleware = serverFields.middleware) == null ? void 0 : _serverFields_middleware.matchers) ? (0, _middlewareroutematcher.getMiddlewareRouteMatcher)((_serverFields_middleware1 = serverFields.middleware) == null ? void 0 : _serverFields_middleware1.matchers) : undefined;
             const interceptionRoutes = (0, _generateinterceptionroutesrewrites.generateInterceptionRoutesRewrites)(Object.keys(appPaths), opts.nextConfig.basePath).map((item)=>(0, _filesystem.buildCustomRoute)('before_files_rewrite', item, opts.nextConfig.basePath, opts.nextConfig.experimental.caseSensitiveRoutes));
+                opts.fsChecker.rewrites.beforeFiles = opts.fsChecker.rewrites.beforeFiles.filter((route)=>!(0, _generateinterceptionroutesrewrites.isInterceptionRouteRewrite)(route));
             opts.fsChecker.rewrites.beforeFiles.push(...interceptionRoutes);
             const exportPathMap = typeof nextConfig.exportPathMap === 'function' && await (nextConfig.exportPathMap == null ? void 0 : nextConfig.exportPathMap.call(nextConfig, {}, {
                 dev: true,
